src/Console/Command.php

```diff
@@ -76,7 +76,7 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
                     $dataType = 'Argument';
                     $suggestionType = 'Values';
                     break;
-                case 'options':
+                default:
```

Comment:
Why would we want to do this? An invalid case as written right now isn't possible; and if there was one I would rather it crashes than it continues silently. I didn't provide a default case on purpose 😉

Reply:
Your logic makes PHPStan cry.

Reply:
@LukeTowers how about this instead? https://github.com/wintercms/storm/commit/35b36b7d686d82bc0eeead5fee1d8f3ee55bbb21

Keeps your `switch` call as is, and appeases PHPStan.

Reply:
But still fails silently. Could do a default that throws an exception I guess.